src/components/Pagination.js

```diff
@@ -0,0 +1,16 @@
+const Pagination = ({currentPage, previousPage, nextPage, perPage, pageLength}) => {
```

Comment:
This is a lot to destructure in the function signature. Maybe consider pulling the prop into the function and destructure there?